lupa/_lupa.pyx

```diff
@@ -1399,12 +1399,22 @@ cdef bint call_python(LuaRuntime runtime, lua_State *L, py_object* py_obj) excep
 
 cdef int py_call_with_gil(lua_State* L, py_object *py_obj) with gil:
     cdef LuaRuntime runtime = None
+    cdef lua_State* stored_state
```

Comment:
Yes, sure, I tried this way first but used None instead of NULL and it obviously failed. Pushed the new one.